recipes/libcurl/all/conanfile.py

```diff
@@ -536,9 +536,9 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "CURL")
-        self.cpp_info.set_property("cmake_target_name", "CURL")
+        self.cpp_info.set_property("cmake_target_name", "CURL::CURL")
         self.cpp_info.set_property("pkg_config_name", "libcurl")
-        self.cpp_info.components["curl"].set_property("cmake_target_name", "libcurl")
+        self.cpp_info.components["curl"].set_property("cmake_target_name", "CURL::libcurl")
         self.cpp_info.components["curl"].set_property("pkg_config_name", "libcurl")
```

Comment:
to remove in conan v2, side effect of `cmake_find_package_*` support

Reply:
Removed. The last changes for `pkg_config_name` define this property as an absolute name, so it'd be overwritten by the root one.